### PR TITLE
Clarify that events are copied into each subscribers input queue

### PR DESCRIPTION
**Pull request by andreasohlund**
Based on https://discuss.particular.net/t/missing-queue-retry-behavior/2353/2

### CODE REVIEW

File: nservicebus/messaging/publish-subscribe/index.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ Depending on the features provided by a given transport there are two possible i
 
 Note: For simplicity these explanations refer to specific endpoints as "Subscribers" and "Publishers". However in reality any endpoint can be both a publisher and/or and a subscriber.
 
+### All subscribers gets a copy of the event
+
+To make sure that each subscriber can process, and potentially retry, the event independent from other subscribers NServiceBus makes sure that each subscriber gets a copy of the published event delivered to their input queue.
```
Comment (danielmarbach) [inline nservicebus/messaging/publish-subscribe/index.md:38]:
```suggestion
To make sure that each subscriber can process, and potentially retry, the event independent of other subscribers NServiceBus makes sure that each subscriber gets a copy of the published event delivered to their input queue.
```

File: nservicebus/messaging/publish-subscribe/index.md [hunk 2]
```diff
@@ -33,6 +33,10 @@ Depending on the features provided by a given transport there are two possible i
 
 Note: For simplicity these explanations refer to specific endpoints as "Subscribers" and "Publishers". However in reality any endpoint can be both a publisher and/or and a subscriber.
 
+### All subscribers gets a copy of the event
```
Comment (danielmarbach) [inline nservicebus/messaging/publish-subscribe/index.md:36]:
```suggestion
### All subscribers get a copy of the event
```

Comment (mikeminutillo) [inline nservicebus/messaging/publish-subscribe/index.md:36]:
```suggestion
### All subscribers gets their own copy of the event
```

**Comment by ramonsmits** [comment]
@andreasohlund nice improvement but I think it can even be better is the gets describes instance vs logical as its an event per logical subscriber and not for each subscription instance. What do you think?

**Comment by andreasohlund** [comment]
We accept PR's ;)

**Pull request merged**